### ISSUE TITLE
Add aria-labels to 'Deploy to Kubernetes' page for improved accessibility

### ISSUE THREAD

**Issue by amisskii** [containers/podman-desktop#8445]
### Is your enhancement related to a problem? Please describe

We need to add aria-labels to the "Deploy to Kubernetes" page because many elements lack them, making navigation and element location challenging.

### Describe the solution you'd like

- Add aria-labels to the relevant page elements.

### Additional context

https://github.com/containers/podman-desktop/issues/5976
